tmt/steps/provision/testcloud.py

```diff
@@ -468,44 +422,84 @@ def start(self) -> None:
 
         # Create instance
         self.instance_name = self._tmt_name()
+
+        # Prepare DomainConfiguration object before Instance object
+        self._domain = DomainConfiguration(self.instance_name)
+        self._domain.memory_size = self.memory * 1024
+        self._domain.cpu_count = 2  # Make configurable
+
+        # Is the combination of host-requested architecture kvm capable?
+        self.kvm = bool(self.arch == platform.machine() and os.path.exists("/dev/kvm"))
+
+        # Is this el <= 7?
+        self.legacy_os = testcloud.util.needs_legacy_net(self._image.name)
+
+        # Is this a CoreOS?
+        self.coreos = bool(re.search('coreos|rhcos', self.image.lower()))
```

Comment:
Do these three names have to be attributes of `self`? They do not seem to be used by any other method, a local scope should be sufficient then.